### ISSUE TITLE
Better appdata summary

### ISSUE THREAD

**Issue by bertob** [diamondburned/dissent#249]
The current summary `GTK4 Discord client in Go (formerly gtkcord4)` isn't in line with the [metadata guidelines](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines) (too long, too technical, doesn't mention what you can do with the app). It's also not the right place to mention gtkcord4, the description would be more appropriate for that.

I'd recommend something like `Chat with your friends on Discord` or just `Chat on Discord`.

**Comment by diamondburned** [comment]
Wouldn't a `Chat with your friends on Discord` or just `Chat on Discord` be a more fitting summary for the official Discord app?

**Comment by diamondburned** [comment]
With the help of Gemini, here are some better ones:

> Option 1: Emphasizing Simplicity and User Focus
> "Lightweight and modern Discord client for Linux"
> 
> Option 2: Highlighting Cross-Platform Potential
> "Native Discord experience for Linux and potentially more"
> 
> Option 3: Focusing on a Unique Feature (If Applicable)
> "Discord client with [unique feature], built for smooth Linux integration"

Option 1 seems best.

**Comment by diamondburned** [comment]
I've updated the repository, README and the metainfo XML file accordingly. Closing this as solved.

**Comment by bertob** [comment]
> Wouldn't a Chat with your friends on Discord or just Chat on Discord be a more fitting summary for the official Discord app?

Not really, because in that case you wouldn't repeat the app name in the summary (they're usually shown together). 

> With the help of Gemini, here are some better ones:

I guess Gemini doesn't know the guidelines too well :D

It's not recommended to use generic adjectives (like "simple" or "fast") or technical details (like "native" or "for Linux"), and focus on what the app actually does for people, e.g. "Manipulate images" rather than "Simple image editor for Linux".

"Lightweight and modern Discord client" isn't terrible, but it is longer than the maximum number of characters (35). Simplifying to e.g. "Lightweight Discord client" would work, but something focused on what you do with the app would be even better :)

**Comment by diamondburned** [comment]
"Discord client alternative" and "Tiny Discord client" (or "Small Discord client") are what I have in mind at the moment. I'm mostly leaning towards "Tiny Discord client", but it's no [terminal client](https://github.com/ayn2op/discordo).

Is the word "client" considered technical?

**Comment by bertob** [comment]
Generally speaking using terms like "client" is not recommended, because it focuses on the protocol/service rather than what it does for people (e.g. chat), but "Tiny Discord client" is passable.

**Comment by diamondburned** [comment]
I thought about "Tiny Discord app", but the guidelines seem to discourage the word "app" specifically. I guess it also avoids the word "client", so I'm not sure what to use here.

**Comment by bertob** [comment]
Well ideally it'd be something focusing on what you do with the app, e.g. "Chat on Discord", but both "Tiny Discord app" and "Tiny Discord client" are fine.

**Comment by diamondburned** [comment]
Based on Vesktop's "Snappier Discord app with Vencord", I think "Tiny Discord app" will be fine.
